src/modules/inspect/task-list/task-list.ts

```diff
@@ -289,20 +289,20 @@ export class TaskList {
       return otherCorrelation.id === correlationId;
     });
 
-    const correlationWasNotFound: boolean = correlation === undefined || correlation === null;
+    const correlationWasNotFound: boolean = correlation === undefined || correlation === null;
```

Comment:
I've always preferred the shorter version of this.

```suggestion
    const correlationWasNotFound: boolean = !correlation;
```

Looks nicer, imho.
Just a suggestion, though.

Reply:
and is more robust as well.